easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -527,6 +539,8 @@ def include_defined_parameters(keyset):
         eb_file.write(('\n'.join(ebtxt)).strip()) # strip for newlines at the end
         eb_file.close()
 
+        self.enable_tamplating = orig_enable_templating
```

Comment:
typo: `tamplating` vs `templating`

this shows you need a better unit test ;)

check whether values containing templates are still being properly 'expanded' after calling dump in the unit test